doc/Language/regexes.pod6

```diff
@@ -282,6 +282,51 @@ Examples of word characters:
     0409 Љ CYRILLIC CAPITAL LETTER LJE
     =end code
 
+=head3 X<C<\c> and C<\C>|regex,\c;regex,\C>
+
+C<\c> takes a parameter delimited by square-brackets which is the name
+of a Unicode character as it appears in the
+L<Unicode Character Database (UCD)|https://unicode.org/ucd/> and matches
+that specific character. For example:
```

Comment:
Maybe simply make a reference to the [literal description](http://docs.p6c.org/language/unicode#Entering_unicode_codepoints_and_codepoint_sequences)

Reply:
I'd like to at least give an example, here. It would also be nice if one or both of these could link to the UCD, which is the canonical source of these names...

Reply:
Please do that. Point is that in the case of \c and \x it's not acting in a way that's different from any other (maybe quoted) character.